### DISCUSSION TITLE
[Projects]: Add `reminder` feature. This could trigger a GitHub notification.

### DISCUSSION THREAD

**Discussion by marcomontalbano** [community/community#67798] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Hi, sometimes I need to be reminded of a card created on the project.
Would be nice to have a kind of `reminder` field type (date + time) that can be used to trigger a notification. It could be a browser/app notification or can be integrated with GitHub notifications.

<img width="100" alt="GitHub notifications" src="https://github.com/community/community/assets/1681269/99a9bf46-2cea-4d9f-92cd-6fe87118262e">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by lidong-linvest21** [comment]
I was looking for that feature for a while, do you find something already?

**Reply by marcomontalbano** [reply]
Unfortunately is not yet implemented. I hope that @github will implement it :pray:

**Reply by lidong-linvest21** [reply]
OK, got you, thanks.
Do you know if there is any other alternatives to realize this reminding function?

**Reply by marcomontalbano** [reply]
I don't know any 😞
